### PR TITLE
Add Storybook to app-project

### PR DESCRIPTION
**Pull request by lancesnider**
Package:

This is a follow up from https://github.com/zooniverse/front-end-monorepo/issues/49#issuecomment-464899047 where @rogerhutchings mentioned wanting to add Storybook. 

This is my first time using Lerna to manage packages, but I think I did it right. :) 

Describe your changes:

- Add Storybook to the app-project
- Configure it so that it looks for *.stories.js files anywhere inside `app-project/src`
- Add Stories for the `shared/Project Statistics/Stat` component
- Add instructions for starting Storybook to README

screen shot 2019-02-19 at 8 52 48 pm

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [x] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
_I've tested on FF, Chrome, and Safari, but don't have a great way to test edge._
- [X] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

File: packages/app-project/src/shared/components/ProjectStatistics/components/Stat/Stat.stories.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Stat from './Stat'
+
+storiesOf('Shared/Components/Project Statistics/Stat', module)
+  .add('Default', () => (
+    <Stat
```
Comment (rogerhutchings) [inline packages/app-project/src/shared/components/ProjectStatistics/components/Stat/Stat.stories.js]: This needs the `Grommet` component wrapping it:

```
import zooTheme from '@zooniverse/grommet-theme'
import { Grommet } from 'grommet'

...

<Grommet theme={zooTheme}>
  <Stat
    ...
  />
</Grommet>
```

File: packages/app-project/src/shared/components/ProjectStatistics/components/Stat/Stat.stories.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      label='Volunteers'
+      value={122}
+    />
+  ))
+  .add('Huge number', () => (
+    <Stat
```
Comment (rogerhutchings) [inline packages/app-project/src/shared/components/ProjectStatistics/components/Stat/Stat.stories.js]: `<Grommet />` again

File: packages/app-project/src/shared/components/ProjectStatistics/components/Stat/Stat.stories.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      label='Volunteers is a long word'
+      value={122000000}
+    />
+  ))
+  .add('Zero', () => (
+    <Stat
```
Comment (rogerhutchings) [inline packages/app-project/src/shared/components/ProjectStatistics/components/Stat/Stat.stories.js]: `<Grommet />` again

**Comment by rogerhutchings** [comment]
Oh, and we have the linter installed now as well, so please run `npm run lint` on your PR - just add your changed files, don't worry if it picks up any others

**Comment by lancesnider** [comment]
Thanks @rogerhutchings. Done. :)

**Comment by rogerhutchings** [comment]
Hi @lancesnider - the top-level `package-lock.json` still needs removing from the PR, but otherwise this is good to go. Thanks!

**Comment by lancesnider** [comment]
@rogerhutchings Done, thanks. :)

**Pull request merged**